PARTICIPANTS.md

```diff
@@ -13,6 +13,6 @@ Person
 Person
 
 3
-
+# Thank you for the git workshop, I found it really useful.
```

Comment:
Can you please say when the workshop was.

Reply:
The workshop held on 27 March, 2018.

Reply:
In the file, please.